teamA/jinwoo/controllers/postController.js

```diff
@@ -0,0 +1,51 @@
+const postService = require('../services/postService');
+
+const postUp = async (req, res) => {
+  try {
+    const { title, content, userId } = req.body;
+
+    if ( !title || !content || !userId ) {
+      return res.status(400).json({ message : 'KEY_ERROR' });
+    }
+
+    await postService.postUp( title, content, userId );
+    return res.status(201).json({ message : 'POSTUP_SUCCESS' });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 500).json({ message : err.message });
+  }
+};
+
+const postList = async (req, res) => {
+    const postsWesta = await postService.postList();
+    return res.status(201).json({ data : postsWesta });
+  }
+
+const postDel = async (req, res) => {
+  try {
+    const postId = req.params.postId;
+    await postService.postDel(postId);
+    return res.status(201).json({ message : 'DELETE_SUCCESS' });
```

Comment:
```suggestion
    await postService.deletePost(postId);
    return res.status(204).json({ message : 'DELETE_SUCCESS' });
```
naming수정에 더해 삭제시 주로 사용되는 No Content의 의미를 담은 204로 수정 할 수 있을 것 같습니다~